Rakefile

```diff
@@ -22,7 +24,7 @@ end
 
 # Ensure the test:integration task behaves as it always has
 desc 'run integration tests'
-task 'test:integration' do |t|
+task 'test:integration' do |_t|
```

Comment:
Can just remove the argument entirely here?

Reply:
Yep